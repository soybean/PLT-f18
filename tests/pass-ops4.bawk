BEGIN {}

LOOP {}

END {
  print(bool_to_string("[0-9]*" % "[0]*"));
  print(bool_to_string("[0-9]*" % "[0-9]*"));
  print(int_to_string(99));
  print(bool_to_string("[0-9]*" !% "[0]*"));
  print(bool_to_string("[0-9]*" !% "[0-9]*"));
  print(int_to_string(99));
  print(bool_to_string("[0-9]*" ~ "[0]*"));
  print(bool_to_string("[0]*" ~ "[0-9]*"));
  print(int_to_string(99));
  print(bool_to_string("[0-9]*" !~ "[0]*"));
  print(bool_to_string("[0-9]*" !~ "[0-9]*"));
}
