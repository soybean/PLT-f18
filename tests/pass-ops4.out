false
true
99
true
false
99
false
true
99
true
false
